WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -1094,6 +1100,22 @@ extension NotificationsViewController: WPTableViewHandlerDelegate {
         return Notification.classNameWithoutNamespaces()
     }
 
+    private var shouldCountNotificationsForSecondAlert: Bool {
+        userDefaults.notificationPrimerInlineWasAcknowledged &&
+            userDefaults.secondNotificationsAlertCount != Constants.secondNotificationsAlertDisabled
+    }
+
+    func tableViewWillChangeContent(_ tableView: UITableView) {
+        guard shouldCountNotificationsForSecondAlert,
+            let note = tableViewHandler.resultsController.fetchedObjects?.first as? Notification,
```

Comment:
Total nitpick - but can this be `notification` (in all the places it's `note`) ? Here it's not too bad, but it is a bit confusing as it's not immediately clear `note` = `notification`.

Reply:
Totally! Note does not mean notification (I don't even recall why I named it that way..)